ui/src/app/views/concept-homepage.tsx

```diff
@@ -327,6 +327,20 @@ export const ConceptHomepage = withCurrentWorkspace()(
         'concepts', 'sets', conceptSet.id, 'actions']);
     }
 
+    get canEdit(): boolean {
+      return this.isOwner || this.isWriter;
+    }
+
+    get isWriter(): boolean {
+      return this.props.workspace.accessLevel as unknown as WorkspaceAccessLevel
+        === WorkspaceAccessLevel.WRITER;
+    }
+
+    get isOwner(): boolean {
```

Comment:
Is it possible to use `WorkspacePermissions` to replace these functions?

I'm not a fan of how `WorkspacePermissions` forces you to construct an object to use its functions but its probably better to reuse than reimplement.

Reply:
yeah i agree.  i may also fix the other place where it's reimplemented to do that too